src/rrule.ts

```diff
@@ -41,6 +31,16 @@ const getnlp: GetNlp = function () {
 // RRule
 // =============================================================================
 
+export const Days = {
+  MO: new Weekday(0),
+  TU: new Weekday(1),
+  WE: new Weekday(2),
+  TH: new Weekday(3),
+  FR: new Weekday(4),
+  SA: new Weekday(5),
+  SU: new Weekday(6)
+}
```

Comment:
Lemme just say for the record I don't love the fact that these enums are duplicated onto the RRule class eg. `RRule.MO, RRule.TU` etc, also `Frequency`. But it would be a breaking change to remove them. Probably a big API overhaul is coming for 3.0.